src/ol_openedx_chat/settings/common.py

```diff
@@ -1,11 +1,12 @@
 # noqa: INP001
+
 """Settings to provide to edX"""
 
 
-def plugin_settings(settings):  # noqa: ARG001
+def plugin_settings(settings):
     """
-    Populate settings
+    Populate common settings
     """
-
-
-DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+    env_tokens = getattr(settings, "ENV_TOKENS", {})
+    settings.OL_CHAT_LLM_MODELS = env_tokens.get("OL_CHAT_LLM_MODELS", [])
+    settings.OL_CHAT_API_KEY = env_tokens.get("OL_CHAT_API_KEY", [])
```

Comment:
Different models will have different keys. We will have to maintain the correct order in lists otherwise we can use the dicts.